src/test/scala/cromwell/engine/backend/CromwellRuntimeAttributeSpec.scala

```diff
@@ -137,7 +143,7 @@ class CromwellRuntimeAttributeSpec extends FlatSpec with Matchers with EitherVal
     val descriptor = WorkflowDescriptor(WorkflowId(UUID.randomUUID()), SampleWdl.WorkflowWithFullGooglyConfig.asWorkflowSources())
     val call = descriptor.namespace.workflow.callByName("googly_task").get
     val unsupported = CromwellRuntimeAttributes.unsupportedKeys(call.task.runtimeAttributes.attrs.keys, BackendType.LOCAL)
-    unsupported shouldEqual Some("Found unsupported keys for backend 'LOCAL': cpu, disks, memory, zones")
+    unsupported shouldEqual Set("disks", "cpu", "zones", "memory")
```

Comment:
Hmm so these keys are allowed for JES backend, and only the JES backend. Keys that are specified but not supported by the backend are filtered out and not validated nor stored in the DB anymore.
This is the reason why I changed those from local to JES because they use "memory" which is only supported by JES.